ui/cypress/integration/login.js

```diff
@@ -0,0 +1,8 @@
+describe('Login test', function () {
+  it('should login', function () {
+    let target_url = Cypress.env('target_url');
+    cy.visit(target_url);
```

Comment:
So, visiting works... Don't you need to set-up anchors for filling the login form? Will it be done in a later ticket?

Reply:
Yep it will be done in this ticket #1107